mne/io/edf/edf.py

```diff
@@ -126,12 +129,13 @@ class RawEDF(BaseRaw):
 
     @verbose
     def __init__(self, input_fname, eog=None, misc=None, stim_channel='auto',
-                 exclude=(), infer_types=False, preload=False, verbose=None):
+                 exclude=(), infer_types=False, preload=False, verbose=None,
+                 include=None):
```

Comment:
```suggestion
                 exclude=(), infer_types=False, preload=False, include=None
                 verbose=None):
```

Reply:
we keep always verbose at the end. It's our convention although I agree it can break some code if verbose param was not named